### PR TITLE
do not allow SUSE Manager downgrade

### PR DESCRIPTION
**Pull request by mbussolotto**
## What does this PR change?

do not allow SUSE Manager downgrade
This is the content of release file:
`SUSE Manager release 5.0 (5.0.1)`
we need to extract `5.0.1`

## Test coverage
- No tests

- [x] **DONE**

## Links

- [x] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

**Comment by mbussolotto** [comment]
it breaks unit tests...I'm fixing it

File: mgradm/shared/utils/exec.go [hunk 1]
```diff
@@ -207,20 +207,21 @@ func SanityCheck(cnx *shared.Connection, inspectedValues *utils.ServerInspectDat
 			)
 		}
 	} else {
-		cnx_args := []string{"s/SUSE Manager release //g", "/etc/susemanager-release"}
-		current_suse_manager_release, err := cnx.Exec("sed", cnx_args...)
+		cnx_args := []string{"s/.*(\\([0-9.]*\\)).*/\\1/g", "/etc/susemanager-release"}
+		b_current_suse_manager_release, err := cnx.Exec("sed", cnx_args...)
```
Comment (cbosdo) [inline mgradm/shared/utils/exec.go:211]:
```suggestion
		b_current_suse_manager_release, err := cnx.Exec("sed", "s/.*(\\([0-9.]*\\)).*/\\1/g", "/etc/susemanager-release")
```

**Review by cbosdo — APPROVED** [review]
Left a cosmetic suggestion, LGTM

**Review by deneb-alpha — APPROVED** [review]
thanks

**Pull request merged**